pkg/scheduler/framework/framework.go

```diff
@@ -354,7 +358,16 @@ func (f *framework) runSchedulingCycleForPickAllPlacementType(
 		return ctrl.Result{}, err
```

Comment:
missed one comment before, once we create the bindings, should we add the finalizer in the crp to make sure all of the bindings are deleted by scheduler before we delete the crp?

Reply:
Oh, that one is added at the upper layer by the scheduler, not the scheduler framework.